lizmap/modules/admin/templates/server_information.tpl

```diff
@@ -37,17 +37,21 @@
     {if empty($modules)}
       <p>{@admin.server.information.no.module@}</p>
     {else}
-        <table class="table table-condensed table-striped table-bordered table-lizmap-modules">
+        <table class="table table-condensed table-striped table-bordered table-server-info table-lizmap-modules">
+        <thead>
         <tr>
             <th>{@admin.server.information.module@}</th>
             <th>{@admin.server.information.module.version@}</th>
         </tr>
+        </thead>
+        <tbody>
         {foreach $modules as $module}
             <tr>
-                <th style='width:20%;'> {$module->slug}</th>
-                <td style='width:20%;'>{$module->version}</td>
+                <th>{$module->slug}</th>
```

Comment:
Not sure why, I took it from L165, it was this.
Before, it was 30% from your initial commit I think.

Reply:
I remove inline style, and prefer to use `table-server-info` which is used in all other tables of this page